### PR TITLE
Add HTTP server output

### PR DESCRIPTION
**Pull request by marc-gr**
- [x] Tests
- [x] Changelog

### CODE REVIEW

File: command/root.go [hunk 1]
```diff
@@ -10,16 +10,17 @@ import (
...
 	"github.com/spf13/pflag"
 	"go.uber.org/multierr"
 	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
 	"golang.org/x/sys/unix"
 
+	"github.com/elastic/stream/pkg/log"
 	"github.com/elastic/stream/pkg/output"
 
 	// Register outputs.
```
Comment (andrewkroh) [inline command/root.go:26]: You'll need to import the `http` output here so that it gets registered.

File: pkg/output/http/http.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/elastic/stream/pkg/log"
+	"github.com/elastic/stream/pkg/output"
+)
+
+func init() {
+	output.Register("httpsrv", New)
```
Comment (andrewkroh) [inline pkg/output/http/http.go:22]: WDYT about calling it and the package `http_server`?

**Review by andrewkroh — COMMENTED** [review]
I tried it out for testing defender_atp. Worked for me. We should discuss how we want pagination and config to work.

**Comment by andrewkroh** [comment]
Not sure if this is related to this output, but it hangs on SIGINT.

```
{"level":"debug","ts":"2021-05-21T08:47:28.666-0400","caller":"command/log.go:74","msg":"Sending log line.","address":"127.0.0.1:8081","log":"alert2.json","line_number":1}
^C^C^C^C^C^C^CError: waiting to write for too long
```

**Comment by marc-gr** [comment]
> Not sure if this is related to this output, but it hangs on SIGINT.
> 
> ```
> {"level":"debug","ts":"2021-05-21T08:47:28.666-0400","caller":"command/log.go:74","msg":"Sending log line.","address":"127.0.0.1:8081","log":"alert2.json","line_number":1}
> ^C^C^C^C^C^C^CError: waiting to write for too long
> ```

Was missing to check the cmd context, this is fixed now 👍

**Comment by marc-gr** [comment]
> I tried it out for testing defender_atp. Worked for me. We should discuss how we want pagination and config to work.

Currently this will just return the given logs in order, one request at a time, so pagination will be defined by the order of the logs you pass to the command. 

This ignores the request URL or any other variable in the request as its initial intent is just to send the stream of logs through an http server output. 

I'd like to have a more sophisticated tool that could be used to mock a specific server, with some routing and assertions, which I think is more in line with what we discussed in https://github.com/elastic/stream/issues/9, I am unsure if this would belong here or as its own separate tool.

From my testing I think this initial approach might be enough for our current needs, WDYT?

**Comment by andrewkroh** [comment]
I think all of that could be part of this tool. I think we should do the minimal required for our first set of test cases and then iterate on the development of features that we need to get the test coverage required. Since it's a test tool and primarily for our own consumption it will be fine if we need to break some of the config to refine it.

**Comment by andrewkroh** [comment]
I noticed the README lists the "outputs". Can you update it to include http-server or remove the list of outputs.

**Pull request merged**